dtfs-central-api/src/v1/email/index.ts

```diff
@@ -0,0 +1,24 @@
+import { AnyObject } from '@ukef/dtfs2-common';
+import { Response } from 'express';
+import externalApi from '../../external-api/api';
+
+export const sendEmail = async (templateId: string, sendToEmailAddress: string, emailVariables: AnyObject) => {
+  try {
+    if (!templateId) {
```

Comment:
Is it even possible for this to happen since `templateId` is a required type/param?

Reply:
This was taken from portal-api which does it

Reply:
Ah, I see that the version of this in portal-api is in JS though, so we don't need this check in here because of TS